container/src/main/java/com/flipkart/poseidon/core/PoseidonServlet.java

```diff
@@ -148,20 +151,26 @@ protected void doRequest(HttpMethod method, HttpServletRequest httpRequest, Http
 
     private void setRequestContext(HttpServletRequest httpServletRequest) {
         RequestContext.set(METHOD, httpServletRequest.getMethod());
-        RequestContext.set(REQUEST_ID, getRequestId(httpServletRequest));
-        RequestContext.set(IS_PERF_TEST, isPerfTest(httpServletRequest));
         RequestContext.set(SOURCE_ADDRESS, httpServletRequest.getRemoteAddr());
     }
 
-    private String getRequestId(HttpServletRequest httpServletRequest) {
-        if (httpServletRequest.getHeader(REQUEST_ID_HEADER) != null) {
```

Comment:
If REQUEST_ID_HEADER, REQUEST_ID, IS_PERF_TEST, PERF_TEST_HEADER constants are not used, can you remove them?